torchgeo/datasets/vhr10.py

```diff
@@ -0,0 +1,537 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
What's the motivation to rename from nwpu to vhr10?

Reply:
NWPU is the name of the institute that created the dataset - Northwestern Polytechnical University. VHR10 is the name of the dataset.

They're also the one that created RESISC45 dataset